LondonBritishLibrary/orient/BLorient7947.xml

```diff
@@ -0,0 +1,164 @@
+                                    <dimensions type="leaf" unit="mm" xml:id="leafdim">
+                                        <height>112-115</height>
+                                        <width>180-183</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>Original pagination by the author from 1 to 454. One leaf after <locus target="#194"/> was omitted from
+                                    numeration.</foliation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout writtenLines="23"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Latin">
+                                <desc>Hand of the author <persName role="author" ref="PRS8943Speedy">
+                                    <roleName type="rank">Captain</roleName>T. C. S. Speedy</persName>.</desc>
+                                <seg type="ink">Black; violet.</seg>
+                                <date notBefore="1867" notAfter="1879"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#1r"/>
+                                    <desc type="ScribalSignature">A note by the author 
+                                        <persName role="author" ref="PRS8943Speedy"><roleName type="rank">Captain</roleName>
+                                            T. C. S. Speedy</persName>.</desc>
+                                    <q xml:lang="en">Vocabulary of the Amharic Language by <persName role="author" ref="PRS8943Speedy">
+                                        <roleName type="rank">Captain</roleName>Tristram Charles Sawyer Speedy</persName>, Interpreter on the
+                                        Staff of <persName ref="PRS7484Napier"><roleName type="title">Lord</roleName> Napier</persName>
+                                        of <placeName ref="LOC4547Maqdal">Magdala</placeName>, during the Expedition to Abyssinia, to release 
+                                        <persName ref="PRS3001Cameron"><roleName type="office">Consul</roleName> Cameron</persName>
+                                        and the European Captives in <date notBefore="1867" notAfter="1868">1867 and 1868</date>, in the reign of the
+                                        <persName ref="PRS9430Tewodros"><roleName type="title">Emperor</roleName> Theodore</persName> 
+                                        of Abyssinia; subsequently Guardian to Theodore's only legitimate Son 
+                                        <persName ref="PRS14505Alamayahu"><roleName type="title">Prince</roleName> Alamayu</persName>
```

Comment:
Maybe you the existing record instead, or however the situation with double records is best solved.

Reply:
Yes, we stay with PRS1625Alamayy.